source/includes/_visitor_events.md

```diff
@@ -197,22 +195,22 @@ https://api.reachlocalservices.com/client_reports/visitor_events?gmaid=TEST_1&vi
 
 |Field Name|Datatype|Description|
 |---|---|---|
-|visitor_id|UUID|Visitor ID|
+|visitor_id|UUID|An id of a visitor to an advertiser’s site for a given browser & device. The same visitor will have a different id if a different browser on the same device or a different device is used.|
 |events|Array|Array of Event Objects. [Event Object](#visitoreventsevent)|
 
 <a name="visitoreventsevent"></a>
 **Event Object**
 
 |Field Name|Datatype|Description|
 |---|---|---|
-|event_id|Event ID|
-|visitor_id|Visitor ID|
-|event_time|Event Time|
-|event_type|Event Type|
-|event_subtype|Event Subtype|
-|referrer_source|Referrer Source|
-|referrer_type|Referrer Type|
-|page_url|Page URL|
+|event_id|Event ID|An id for the event and unique to every event
```

Comment:
Actually, not unique to every event.  I think if they are within a certain small time span, it's the same.  "same visit" perhaps?

Reply:
event_id is a GUID and is unique. In fact, it's the primary key of the events table.

Reply:
need a | at the end here also

Reply:
[event_id + occurrence_time] are the unique primary key.  There are lots of duplicate event_ids in the table.